upgrade.md

```diff
@@ -52,11 +52,20 @@ Theoretically we can do
 
 to update all apps. This will possibly work, but it's not guaranteed and will be a bit tedious to recover from. After a major upgrade I worked through the list of apps and categorised problems - some were due to missing libraries on the server due to the upstream docker image changing, some where deploy keys that had been removed.
 
+To provision an app, use the `./scripts/provision` script, such as
+
 ```
-./scripts/provision msc-idm-2019
 ./scripts/provision msc-idm-2020
 ```
 
+and then check the staging server, as http://129.31.x.y:3838/staging/msc-idm-2020/
```

Comment:
So once we've checked the staging server, is there a further step to release the app?

Reply:
Set up this way, this was running on my desktop, so there was no release. But this meant that I could test the system without having the server down all day. Once I'd iterated through a couple of times (build, deploy, build, deploy) and confirmed all applications worked, I then did a much quicker deploy on the real machine the normal way

Reply:
Ah got you, this is just for testing somewhere, cool